### PR TITLE
serializer with polygon

### PR DESCRIPTION
**Pull request by rebert**

### CODE REVIEW

**Comment by rebert** [comment]
@hansmannj für ein sauberes migrations hat es nicht gereicht

**Comment by hansmannj** [comment]
This should work now, thanks for your work on that @rebert 
Probably there should be some more unit tests on it (in another PR).
And there is an issue, that the calculated bbox from the code differs from the expected one in the sample item, from which the geometry was copied. But I think, this might as well be an error in the sample data. To be checked on Monday.

**Review by ltshb — CHANGES_REQUESTED** [review]

**Comment by ltshb** [comment]
What happened with the migration files ? Why do you change existing ? Why there is two migration starting with 0017 ?

**Comment by ltshb** [comment]
Before merging, we need to first wait on PR #73 and #74 to be merged. Then recreate the migrations file for this PR.

File: app/stac_api/serializers.py
Comment (rebert) [reply app/stac_api/serializers.py:171]: In theory we will never need to try except this... it is more to have a easier live when developing... when deleting the bbox of a item by purpose in the db

File: app/stac_api/models.py
Comment (ltshb) [inline app/stac_api/models.py]: This extent_geometry should not be editable by the user and it should be `blank=True` otherwise it is added to the admin page.

**Review by ltshb — APPROVED** [review]
:+1:

**Comment by rebert** [comment]
@hansmannj thanks for your support.

**Pull request merged**
